daemon/src/main/java/org/apache/maven/cli/DaemonMavenCli.java

```diff
@@ -808,7 +822,10 @@ private void logSummary(ExceptionSummary summary, Map<String, String> references
 
     private static final String ANSI_RESET = "\u001B\u005Bm";
 
-    private void configure(CliRequest cliRequest)
+    private static void configure(
+            CliRequest cliRequest,
+            EventSpyDispatcher eventSpyDispatcher,
+            Map<String, ConfigurationProcessor> configurationProcessors)
```

Comment:
Same as above...
Those methods are private and need to be given fields, so it's better to make them non static imho.

Reply:
configure is called twice with different params, so it is IMO safer not to allow reading the instance fields.